### ISSUE TITLE
dataset download issue

### ISSUE THREAD

**Issue by HBioquant** [OptiMaL-PSE-Lab/DeepDock#16]
Hi,
I found that the downloaded training set and test set data are damaged by the data/get_deepdock_data.sh. Can you update the data set again?
Thanks.
David.

**Comment by 1874Mercury** [comment]
> Hi, I found that the downloaded training set and test set data are damaged by the data/get_deepdock_data.sh. Can you update the data set again? Thanks. David.


Hi, I meet the same problem. Did you fix it?
